### ISSUE TITLE
Methane/N2O lifetime modelling decisions

### ISSUE THREAD

**Issue by njleach** [OMS-NetZero/FAIR-pro#13]
-So this is a real issue at the moment that has come up due to the current main usage of my FAIR version- the CO2-fe metric.

-I recently changed the way I calculate CH4 and N2O concentrations to be more MAGICCal, in that now the entire abundance of atmospheric gas decays rather than just the perturbation on top of a pre-industrial concentration. This is exactly what MAGICC does, and seems to make sense physically, but had the issue that for this to work we require TOTAL emissions to be provided, rather than just anthropogenic emissions (as are provided in the RCP database). 

-To determine example natural emissions, I backed out total emissions from the model, taking the RCP concentrations to be "absolute". I then subtract off the RCP emissions to find what we've been calling the "natural" emissions, which get added on to the RCP ones when we run this version of FAIR.

-This way of doing things is, I believe, exactly what MAGICC has done, and indeed, using these natural+RCP emissions reproduces both the RCP concentrations and  RFs pretty much perfectly (as you'd hope since I backed them out to work out the emissions- very circular I know).

-However, this causes major issues when trying to determine CO2-fe emissions since the "natural" emissions are not flat, as we'd probably hope- they're quite close, and in the case of N2O, the decrease does fit with literature, but unfortunately, when we use a differencing method to calculate the RF due to an individual species (CH4 causes most problems)- ie. calculate RF including all emissions then remove the anthropogenic component for that species and difference the two- due to the now NOT flat natural baseline, the RF difference calculated is not close to the RCP RF (30% out). Another reason to be sceptical of these natural emissions is- although they are what's required to reproduce the RCP concentrations from RCP emissions- the natural methane emissions decrease from around 260 Mt/yr to 200 Mt/yr, which is far more than they have ever done in the past 1000 years based on ice core data. Although this could be attributable to anthropogenic influence on natural habitats increasing during the industrial revolution and beyond, it's unlikely that this would cause such a major change to the **total** natural emissions which have been pretty much flat since 1000, and probably before.

-Richard reckons I should try and go for a more IPCC method compared to MAGICC and have two separate lifetimes for each gas species- a **natural** one for the baseline abundance to decay over and a **perturbation** lifetime for everything above it to decay, and then see what natural emissions would be backed out of that, again using the RCP concentrations as correct. Hopefully they will be flat.

**Comment by njleach** [comment]
![image](https://user-images.githubusercontent.com/29585862/28732391-993eed22-73cf-11e7-996b-420ffc716516.png)

The dotted lines in the 2nd and 3rd row left hand side panels show the natural emissions. The N2O ones follow literature encouragingly well, but CH4, despite looking like a small change compared to anthropogenic, reduces far too much relative to history at the moment.

**Comment by znicholls** [comment]
@njleach 

Only some quick comments from me so apologies for any gaps. My concern at the moment is why we're going for such a MAGICC like model. If we're just going to end up reproducing MAGICC, why are we going to all this effort? 

Re backing out natural emissions, as you say your current method is very circular. If you would like natural emissions from MAGICC, I have them so can send them through as part of the outputs of the runs I owe you.

**Comment by richardcode** [comment]
@njleach Thanks for this Nick. I'll try to have a read up on methane simple models over the weekend, so I'm more up to speed with the options for next week and we can discuss strategies then. 

@znicholls MAGICC natural methane (and N2O) emissions would be great if it is simple to get. Looking at the documentation it seems that they are derived as the average over a ten-year period as the residual to close the atmospheric budget when best estimate anthro emissions are used. Assume they are then held constant in the future (and also for periods previous?)?

**Comment by njleach** [comment]
@znicholls 
@richardcode 
Yep- I can completely see your concern. Currently, because the CH4 and N2O lifetime models in MAGICC are fairly simplistic anyways, there's not a huge amount I can do differently from MAGICC. I changed my concentration calculations to be much more like MAGICC as I was unable to reproduce the N2O RCP concentrations at all closely when using RCP emissions when using my original (perturbation only) model, so thought I'd try and bring it more in line with MAGICC- the advantage of mine being that it is still a little simpler (keeping CH4 lifetime dependent on only temperature and its own abundance whereas MAGICC includes other aerosol factors), and that it is much more clear what I'm doing, and on GitHub (I did have real trouble working out exactly what MAGICC itself was doing and had to do quite an in depth literature search). 

I'm now repeating what Richard has commented (beat me to it!) but yes, natural emissions as used by MAGICC would be fantastic to have! 

With the natural emissions from MAGICC, I can try both my simplified CH4 MAGICC lifetime model, and my original model (that had linear feedbacks on both abundance and temperature) to see if something like that is accurate enough- that would be ideal as it's as simple as we can go!

**Comment by njleach** [comment]
So to add and fill you guys in on what I've been up to: 

The MAGICC outputs that you sent us @znicholls were very useful in seeing what MAGICC actually does with the RCP emissions. As far as myself and @richardcode could tell, it seems for a normal run MAGICC outputs the RCP concentrations until 2005 and then switches to calculating concentrations from emissions? The runs that showed large deviations from the RCPs in the past seemed to be what MAGICC actually calculates from the emissions in the past. This represented a problem as before the goal had been to get a model that outputs the RCP concentrations from the RCP emissions- but I now think that without doing a "natural emissions" trick (adding natural emissions that are what they need to be to make up the concentrations) it's pretty much impossible to match the emissions in the past AND the future. We've now decided to back out emissions from concentrations using my current methane/ N2O models and check if they are within ensemble results (there's a Prather paper from 2016 on something similar).

This week I've also been working on other Montreal/Kyoto gases, and this has been another problem. I again started by matching the RCP emissions to their concentrations as well as possible, which I could to a reasonable degree of accuracy. I had two feedbacks in- one temperature dependent and one methane concentration dependent. Ideally I'd like to be able to simplify this down to just a temperature feedback as having interdependency within the model reduces its flexibility substantially. Then I hit another issue, when trying to match CMIP6 concentrations- the RCP emissions profiles are just not compatible in some places with CMIP concentrations (analogous to the RCP concentrations disagreeing substantially with CMIP6 concentrations). I think therefore I may try and wait until CMIP releases emissions data (which is likely to be extremely useful for these species as their emissions history is generally much more recent than, for example, CH4), and then match THAT emissions data to the CMIP6 data, while keeping sensitivity coefficients within reasonable levels such that for extreme scenarios the future concentrations don't do crazy things.

Hopefully this all makes sense! Let me know if you have any comments/ suggestions, but I think that the backing out emissions approach should help us to compare my models to some recent papers more easily.

**Comment by znicholls** [comment]
Looks great @njleach. I think you and @richardcode have the right idea re MAGICC's behaviour. 

Re Kyoto gasses I think you've read this correctly. The CMIP6 concentrations (and emissions when they appear) are a fairly major update on the RCPs. Only thing I'd say is that I wouldn't get too worried about trying to model extreme scenarios. If we get to that stage climate models might not be much use anyway...
